eve/main.yml

```diff
@@ -216,7 +224,6 @@ stages:
           command: >
             sudo mount -o loop metalk8s.iso
             /srv/scality/metalk8s-%(prop:metalk8s_version)s
-          haltOnFailure: true
```

Comment:
I think we should always keep `haltOnFailure: true`.. no??

Reply:
Agree